docs/decisions/0003-learner-credit-management.rst

```diff
@@ -0,0 +1,37 @@
+3. Learner Credit Management
+============================
+
+Status
+******
+
+Accepted - July 2022
+
+Context
+*******
+
+* We decided to enable the consumption of Enterprise Offers in the Enterprise Learner Portal (TODO: link to LP ADR)
+* Enterprise admins need the ability to check the cash remaining on their Enterprise offer,
```

Comment:
The Learner Credit Management screen also currently supported offers with no `max_discount` set, in which case there is no notion of "cash remaining" or upper limit. Instead a single card with "Total Redeemed" is shown to show their current spend.

Might be worth expanding the context to include this bit.